sdk/core/azure-core/src/http/winhttp/win_http_transport.cpp

```diff
@@ -12,7 +12,9 @@
 
 #if defined(BUILD_TRANSPORT_WINHTTP_ADAPTER)
 #include "azure/core/http/win_http_transport.hpp"
+#include "win_http_request.hpp"
```

Comment:
```suggestion
#include "azure/core/http/win_http_request.hpp"
```

Reply:
win_http_request is in the same directory as win_http_transport.cpp. So that would have to be something different. Theoretically I could put it in a "private" subdirectory but it would be rather lonely there.

Reply:
Oh I see, ignore my comment.  Didn't realize this is a private header.